### ISSUE TITLE
EVAL_COUNT fixed as 1000

### ISSUE THREAD

**Issue by craigdsthompson** [ubc-tuehoang/testScript#1]
https://github.com/ubc-tuehoang/testScript/blob/dcfe6aea6b27ce8693c25175bfb1f44f160ee551/concurrent_curl1-4.c#L9

Can you clarify why EVAL_COUNT is 1000? I would assume that we'd have to get the number of tokens from the api response and that it would be different for each call.

**Comment by craigdsthompson** [comment]
@ubc-tuehoang you mentioned this in our meeting today, but I'm not sure I followed. Could you please reiterate what you said here as well so that we can discuss?

**Comment by ubc-tuehoang** [comment]
@craigdsthompson - 
The objective of this test is to measure the response time for each thread call (from start to completion) independently of the LLM engine used. The focus is solely on the completion time of each thread. The EVAL_PARM is a control value that can invert the graph.

**Comment by craigdsthompson** [comment]
The graphs you shared in slack have "Tokens/s" on the y-axis. Did you use this script to generate the data for these graphs?

**Comment by ubc-tuehoang** [comment]
@craigdsthompson - yup, the C file performs the load test and generates the data points; and I just added the python script to create the graph.

**Comment by craigdsthompson** [comment]
I'm wondering if the plots shared in slack might be misleading? It looks to me like each data point represents `1000/response_time` but is labeled as `Tokens/s`.

If the API call returns exactly 1000 tokens, then the label is accurate. But, if the API call actually returned 200 tokens, then the datapoint plotted would be off by a factor of 5x. Is that right? Or am I misinterpreting somehow?

**Comment by ubc-tuehoang** [comment]
you are correct- it should have not labled it as tokens/s.  The EVAL_PARM is a control value for inverting the graph only.
